swaglyrics/cli.py

```diff
@@ -109,14 +109,16 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
 		with open(unsupported_txt, 'a', encoding='utf-8') as f:
 			f.write(f'{song} by {artist} \n')
 			f.close()
-		if make_issue:
+		if make_issue and re.sub(aln, '', song + artist) != song+artist:  
```

Comment:
A match solution is preferable to a substitution one

Reply:
How about search?